python/py-buildbot-macports-custom-views/Portfile

```diff
@@ -20,7 +20,7 @@ homepage            https://github.com/macports/buildbot-macports-custom-views
 master_sites        pypi:[string index ${python.rootname} 0]/${python.rootname}
 distname            ${python.rootname}-${version}
 
-checksums           sha256  fc40e83b45547e0bc89135cd9618e322ff4b3fb27a16384c71cbddc3c8877515 \
+checksums           sha256  ce2d4ecbefdc0d8f5d4a7087ac1c7dcb45fa6bdd045bef8699966bc0dd6b5519 \
                     rmd160  5de014cca752bc35d64cc791862fd0967ee0556b \
                     size    578249
```

Comment:
It seems impossible that the sha256 checksum changed and the rmd160 did not. The CI seems to agree.